cprover_bindings/src/irep/to_irep.rs

```diff
@@ -253,6 +253,11 @@ impl ToIrep for ExprValue {
                 ],
             },
             ExprValue::Nondet => side_effect_irep(IrepId::Nondet, vec![]),
+            ExprValue::Poison => {
+                // For  now, Poison is lowered to Nondet with a comment.
+                // In the future, CBMC might handle poison expressions directly
+                side_effect_irep(IrepId::Nondet, vec![]).with_comment("deinit")
```

Comment:
Is "deinit" always the comment we want? What about cases where Kani might introduce poison itself? Why not just use "poison" as the comment?

Reply:
Yes actually, I moved that code from the previous version of the pr and should have changed the name